docs/WindowsBuild.md

```diff
@@ -0,0 +1,139 @@
+# Building Swift on Windows
+
+This document describes how to build Swift for Windows natively. For more information about Swift on Windows, see [the Windows doc](./Windows.md) for more context of
+what's possible on Windows.
+
+There are two supported ways to build Swift on Windows, they are
+
+1. Using [`clang-cl`](https://clang.llvm.org/docs/UsersManual.html#clang-cl)
+1. Using Microsoft Visual Studio Code (MVSC)
+
+There is a limitation to building MVSC, the Swift Runtime can't be 
+built. So though it's possible to build the compiler and the stdlib, and use 
+built products to compile a Swift program, it won't be possible to run the 
+binary without seperately obtaining the Swift Runtime. So the `clang-cl` method
```

Comment:
Again: don't start the sentence with 'so'; if you want to do something like this, start with 'therefore' or 'as such'. 

With that said, I'd move this sentence up to the start of the paragraph: "Using `clang-cl` is recommended over MSVC for building on Windows.", and then you go into the reasons why.